src/scripts/main.js

```diff
@@ -1 +1,68 @@
 'use strict';
+
+let right;
+let left;
+
+function display1(order) {
+  const box1 = document.createElement('div');
+
+  document.body.appendChild(box1);
+  box1.textContent = `${order} promise was resolved`;
+}
+
+function display2(order) {
+  const box2 = document.createElement('div');
+
+  document.body.appendChild(box2);
+  box2.textContent = `${order} promise was rejected`;
+}
```

Comment:
Both functions do almost the same. Try to combine them and don't forget to add the correct class for message
![image](https://user-images.githubusercontent.com/96409563/225556017-6ae2e10a-ae05-4dcd-84e7-07bd7db19718.png)